clearwater_config_manager/scripts/upload_fallback_ifcs_xml

```diff
@@ -11,6 +11,7 @@ FILENAME=/etc/clearwater/fallback_ifcs.xml
 SCHEMA=/usr/share/clearwater/clearwater-config-manager/scripts/config_validation/fallback_ifcs_schema.xsd
 local_site_name=site1
 etcd_key=clearwater
+apply_fallback_ifcs=Y
```

Comment:
It was meant to be so that apply_fallback_ifcs/request_shared_ifcs were always set to a negative value even if they're not set at all in the config. This then makes the checking later simpler.
I should have set these to N rather than Y though :)